lib/mastodon/media_cli.rb

```diff
@@ -44,6 +44,88 @@ def remove
       say("Removed #{processed} media attachments (approx. #{number_to_human_size(aggregate)}) #{dry_run}", :green, true)
     end
 
+    option :start_after
+    option :dry_run, type: :boolean, default: false
+    desc 'remove-orphans', 'Scan storage and check for files that do not belong to existing media attachments'
+    long_desc <<~LONG_DESC
+      Scans file storage for files that do not belong to existing media attachments. Because this operation
+      requires iterating over every single file individually, it will be slow.
+
+      Please mind that some storage providers charge for the necessary API requests to list objects.
+    LONG_DESC
+    def remove_orphans
+      progress        = create_progress_bar(nil)
+      reclaimed_bytes = 0
+      removed         = 0
+      dry_run         = options[:dry_run] ? ' (DRY RUN)' : ''
```

Comment:
maybe we should make this :destructive so it's opt in instead of opt out?

Reply:
or maybe we should gather all the files to delete, ask for confirmation, and then delete them

Reply:
I wouldn't do that because of memory implications if there are a lot of files as well as being able to interrupt the process while having some kind of tangible progress.